### ISSUE TITLE
<collection>.remove() with array of keys or documents

### ISSUE THREAD

**Issue by neunhoef** [arangodb/arangojs#349]
In 3.0 we have introduced that the DELETE endpoint /_api/document/<collection> can be used to delete multiple documents (see https://docs.arangodb.com/3.1/HTTP/Document/WorkingWithDocuments.html#removes-multiple-documents). In the server JS and in `arangosh` this is implemented in the way that the .remove() method of a collection object can take a list of strings (keys or ids) or a list of documents and will then remove all the documents given. It would be desirable to implement this behaviour in `arangojs` as well.

**Comment by rraziel** [comment]
I've checked the [sources](https://github.com/arangodb/arangojs/blob/master/src/collection.ts#L249) and it seems like this is not implemented yet.
Would it be ok if I worked on this and submitted a pull request?
Are there any guidelines/recommendations/etc.?

**Comment by rraziel** [comment]
Actually just noticed the `removeByKeys`, so maybe it's already in there?

**Comment by pluma** [comment]
No, this is separate from remove-by-keys, which uses the deprecated simple query API.

@rraziel if you'd like to work on this, that'd be awesome. Just keep in mind we need a signed Contributor License Agreement (CLA) for legal reasons before we can merge any PR.

You can download the form from our website: https://www.arangodb.com/documents/cla.pdf

Once you've filled out and signed it, please send it to us via e-mail: cla@arangodb.com or fax: +49-221-2722999-88.

**Comment by pluma** [comment]
This issue will be addressed by the upcoming major release.
